help/en/package/jmri/jmrit/withrottle/UserInterface.shtml

```diff
@@ -77,11 +77,11 @@
 					<th>Web</th>
 				</tr>
 				<tr>
-					<td><a target=_new href="https://withrottle.com">WiThrottle</a></td>
+					<td><a target=_blank href="https://withrottle.com">WiThrottle</a></td>
```

Comment:
Shouldn't these normally be enclosed in quotes?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Link_that_opens_a_new_tabwindow

Reply:
My understanding is that they're optional in this HTML attribute, since the value has no spaces or other special characters.
However, I see that several people online recommend using them, so I'll add them. No worries.